docs/FAQ/how-deploy-bold-bi-in-eks-using-network-load-balancer-with-ssl-certificate-from-acm.md

```diff
@@ -0,0 +1,245 @@
+# How to deploy Bold BI in EKS Using Network Load Balancer (NLB) with SSL certificate from AWS Certificate Manager (ACM).
+
+The below section explains how to deploy Bold BI in EKS using Nginx Network Load Balancer (NLB) with SSL certificate from AWS Certificate Manager (ACM).
+
+## Create a EKS Cluster and filestorage:
+
+1. Create an Amazon EKS cluster and [node group](https://docs.aws.amazon.com/eks/latest/userguide/eks-compute.html) to deploy Bold BI.
+
+   https://docs.aws.amazon.com/eks/latest/userguide/getting-started.html 
+
+2. Create an Amazon Elastic File System (EFS) volume to store the shared folders for application usage by following the below link.
```

Comment:
2. Create an Amazon Elastic File System (EFS) volume to store the shared folders for application usage by following the link below.

Reply:
concern addressed